test/built-ins/Atomics/wake/symbol-count-throws.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018 Rick Waldron.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-atomics.wake
+description: >
+  Allowed boundary cases of the third 'count' argument to Atomics.wake
```

Comment:
I realize this is nitpicking -- but this description makes not sense to me. Write "Return abrupt when symbol passed as 'count' agrument to Atomics.wake"

Reply:
Agreed. I will fix.